cookbook/gestures/dismissable.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
 title: "Implement Swipe to Dismiss"
-permalink: /cookbook/gestures/dismissible/
+permalink: /cookbook/gestures/dismissable/
```

Comment:
Could we please keep this as `dismissible`? I thought it was `dismissable` as well, haha, but it turns out that's actually not a word!  😄 

https://docs.flutter.io/flutter/widgets/Dismissible-class.html